bundles/org.openhab.binding.etherrain/src/main/java/org/openhab/binding/etherrain/internal/api/EtherRainCommandResult.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.etherrain.internal.api;
+
+/**
+ * The {@link EtherRainCommandResult} is the response packet for Command Result
+ *
+ * @author Joe Inkenbrandt - Initial contribution
+ */
+public enum EtherRainCommandResult {
+    RESULT_OK("OK"),
+    RESULT_INTERRUPTED_RAIN("RN"),
+    RESULT_INTERUPPTED_SHORT("SH"),
+    RESULT_INCOMPLETE("NC");
+
+    protected String result;
```

Comment:
For an enum I would expect this `private `and `final`, please apply to all applicable `enum`s.